diff --git a/build/gulpfile.reh.ts b/build/gulpfile.reh.ts
index c643cf99..ef3dffa7 100644
--- a/build/gulpfile.reh.ts
+++ b/build/gulpfile.reh.ts
@@ -329,2 +329,4 @@ function packageTask(type: string, platform: string, arch: string, sourceFolderN
 			.pipe(jsonEditor((json: Record<string, unknown>) => {
+				json.cliDownloadUrlTemplate = 'https://github.com/!!ASSETS_REPOSITORY!!/releases/download/!!RELEASE_VERSION!!/!!APP_NAME_LC!!-cli-${os}-${arch}-!!RELEASE_VERSION!!.tar.gz';
+				json.serverDownloadUrlTemplate = 'https://github.com/!!ASSETS_REPOSITORY!!/releases/download/!!RELEASE_VERSION!!/!!APP_NAME_LC!!-reh-${os}-${arch}-!!RELEASE_VERSION!!.tar.gz';
 				json.commit = commit;
diff --git a/build/gulpfile.vscode.ts b/build/gulpfile.vscode.ts
index 95ae1790..c7736297 100644
--- a/build/gulpfile.vscode.ts
+++ b/build/gulpfile.vscode.ts
@@ -201,2 +201,4 @@ function packageTask(platform: string, arch: string, sourceFolderName: string, d
 			.pipe(jsonEditor((json: Record<string, unknown>) => {
+				json.cliDownloadUrlTemplate = 'https://github.com/!!ASSETS_REPOSITORY!!/releases/download/!!RELEASE_VERSION!!/!!APP_NAME_LC!!-cli-${os}-${arch}-!!RELEASE_VERSION!!.tar.gz';
+				json.serverDownloadUrlTemplate = 'https://github.com/!!ASSETS_REPOSITORY!!/releases/download/!!RELEASE_VERSION!!/!!APP_NAME_LC!!-reh-${os}-${arch}-!!RELEASE_VERSION!!.tar.gz';
 				json.commit = commit;
